### PR TITLE
add statistics entrypoint

### PR DESCRIPTION
**Pull request by touv**

### CODE REVIEW

File: services/data-computer/v1/statistics.ini [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Entrypoint output format
+mimeType = application/json
+
+# OpenAPI Documentation - JSON format (dot notation)
+post.operationId = post-v1-statistics
+post.summary = Chargement et analyse d'un fichier corpus
+post.description = Le corpus est analysé et pour chque valeur envoyée (champ value), le service retourne quelques statistiques.
```
Comment (parmentf) [inline services/data-computer/v1/statistics.ini:7]: **typo:** Un tantinet de français + MarkDown‏
```suggestion
post.description = Le corpus est analysé et pour chaque valeur envoyée (champ `value`), le service retourne quelques statistiques.
```

File: services/data-computer/v1/statistics.ini [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+[env]
+path = generator
+value = statistics
+
+# Step 1 (générique): Charger le fichier corpus
+[delegate]
+file = charger.cfg
+
+# Step 2 (générique): Traiter de manière asynchnore les items reçus
```
Comment (parmentf) [inline services/data-computer/v1/statistics.ini:43]: **polish:** ‏Let's fix comments ;)
```suggestion
# Step 2 (générique): Traiter de manière asynchrone les items reçus
```

File: services/data-computer/v1/statistics.ini [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+[fork]
+standalone = true
+logger = logger.cfg
+
+# Step 2.0 (optionnel): Accélére le détachement du fork si l'enrichissement est lent
```
Comment (parmentf) [inline services/data-computer/v1/statistics.ini:48]: **polish:** ‏Derechef
```suggestion
# Step 2.0 (optionnel): Accélère le détachement du fork si l'enrichissement est lent
```

File: services/data-computer/v1/statistics.ini [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+value = get('value')
+
+[fork/delegate/replace]
+path = id
+value = get('id')
+
+path = value
+value = get('_statistics.value')
+
+# Step 2.2 (générique): Enregister le résulat et signaler que le traitment est fini
```
Comment (parmentf) [inline services/data-computer/v1/statistics.ini:73]: **polish:** ‏Fix comments
```suggestion
# Step 2.2 (générique): Enregistrer le résultat et signaler que le traitement est fini
```

**Review by parmentf — CHANGES_REQUESTED** [review]
**Warning**, [Dockerfile](https://github.com/Inist-CNRS/web-services/blob/services/data-computer/add-stats/services/data-computer/Dockerfile) in this branch is *not production-ready* (it does not yet use the last `ezs-python-server` and still have a `USER daemon` line).

The reason is, there is another PR that implements it, but is not yet merged: #64 
See its [Dockerfile](https://github.com/Inist-CNRS/web-services/blob/services/data-computer/doc-lda/services/data-computer/Dockerfile).

**Comment by touv** [comment]
> **Warning**, [Dockerfile](https://github.com/Inist-CNRS/web-services/blob/services/data-computer/add-stats/services/data-computer/Dockerfile) in this branch is _not production-ready_ (it does not yet use the last `ezs-python-server` and still has a `USER daemon` line).
> 
> The reason is, there is another PR that implements it, but is not yet merged: #64 See its [Dockerfile](https://github.com/Inist-CNRS/web-services/blob/services/data-computer/doc-lda/services/data-computer/Dockerfile). See #68

how to get out of this depthlock  ?

**Comment by parmentf** [comment]
> > **Warning**, [Dockerfile](https://github.com/Inist-CNRS/web-services/blob/services/data-computer/add-stats/services/data-computer/Dockerfile) in this branch is _not production-ready_ (it does not yet use the last `ezs-python-server` and still has a `USER daemon` line).
> > The reason is, there is another PR that implements it, but is not yet merged: #64 See its [Dockerfile](https://github.com/Inist-CNRS/web-services/blob/services/data-computer/doc-lda/services/data-computer/Dockerfile). See #68
> 
> how to get out of this depthlock ?

I think we could improve the Docker in this PR.
It should be the same Dockerfile as in #64, and so no conflicts should arise.

**Comment by parmentf** [comment]
It would be better with an example to test.

**Comment by touv** [comment]
> It would be better with an example to test.

à mettre ici https://github.com/Inist-CNRS/web-services/blob/main/services/data-computer/tests.hurl#L40
;-)
I will do it asap

File: services/data-computer/tests.hurl [hunk 5]
```diff
@@ -37,4 +37,39 @@ delay: 1000
...
+variable "computing_token" exists
+
+# There should be a waiting time, representing the time taken to process data.
+# Fortunately, as the data is sparse, and the computing time is small,
+# the need is small.
+
+# Version 4.1.0 of hurl added a delay option, which value is milliseconds.
+# https://hurl.dev/blog/2023/09/24/announcing-hurl-4.1.0.html#add-delay-between-requests
+
+POST https://data-computer.services.istex.fr/v1/retrieve-json
```
Comment (parmentf) [inline services/data-computer/tests.hurl:60]: **issue:** ‏You forgot the `host` variable
```suggestion
POST {{host}}/v1/retrieve-json
```

**Review by parmentf — CHANGES_REQUESTED** [review]

**Comment by parmentf** [comment]
Ouch. The previous test fails now (tree-segment).

**Pull request merged**